src/js/themes/base.d.ts

```diff
@@ -946,6 +951,25 @@ export interface ThemeType {
     colors?: GraphColorsType;
     extend?: ExtendType;
   };
+  pagination?: {
+    container?: {
+      extend: ExtendType;
+    };
+    controls?: {
+      align?: AlignContentType;
+      direction?: DirectionType;
+      gap?: GapType;
+      pad?: PadType;
+      margin?: MarginType;
+    };
+    icons?: {
+      color: ColorType;
+      first: React.ReactNode;
+      last: React.ReactNode;
+      next: React.ReactNode;
+      previous: React.ReactNode;
```

Comment:
I don't think all these icons have to be defined. You might be missing the question marks symbol to indicate those are optional.

Reply:
Done.